deploy/.env

```diff
@@ -17,4 +17,4 @@ WORKFLOW_DASHBOARD_PORT=5200
 WEBSOCKET_SVC_PORT=3500
 KYB_APP_PORT=5201
 DOMAIN_NAME=""
-HASHING_KEY_SECRET="secret"
+HASHING_KEY_SECRET_BASE64=JDJiJDEwJDNFeWtwWEs4QkdiczlRaWFwLkM4Vk8=
```

Comment:
When initializing the monorepo there is a script that generate new salt and transform it to base64 than update .env with the value in order to prevent similar envs with same secrets.

would we like also to replace the hashing key in here ?

Reply:
Added those changes